res/test/provider.json

```diff
@@ -14,6 +14,6 @@
         "version": "14.78.45.31"
     },
     "serviceConfiguration": {
-        "applicationDirectory": "http://localhost:3923/provider/sample-app-directory.json"
+        "applicationDirectory": "http://localhost:3923/test/sample-app-directory.json"
```

Comment:
If making this change, the hook that overrides the previous URL should no longer be required - can we remove it?